src/__test__/breadcrumbstest.spec.tsx

```diff
@@ -1,13 +1,14 @@
-const { Builder, By} = require('selenium-webdriver')
-const {browser, sleep, n, options} = require("../setupTests");
-
 describe('Default Suite', function() {
   let driverCrumbs
+  var webdriver = require("selenium-webdriver");
+  var By = webdriver.By;
+  var {browser, sleep, n, options} = require("../setupTests");
+
   beforeAll( async function() {
     await sleep(n);
   })
   beforeEach(async function() {
-    driverCrumbs = await new Builder().forBrowser(browser).setFirefoxOptions(options).build();
+    driverCrumbs = new webdriver.Builder().forBrowser(browser).setFirefoxOptions(options).build()
```

Comment:
I am not sure if this is the right place. We should not have to instantiate a new webdriver for every test. This slows down test execution a lot. However I am not sure what's best practice.

Reply:
But first we'll have to fix all tests before we refactor.